pkg/spec/resources.go

```diff
@@ -669,9 +669,10 @@ func parsePortMapping(pm string) (*api_v1.ServicePort, error) {
 	// Case 3 - port/protocol
 	// Case 4 - port:targetPort/protocol
 	case 2:
-		switch api_v1.Protocol(protocolSplit[1]) {
+		protocol_uppercase := strings.ToUpper(protocolSplit[1])
```

Comment:
just one small nitpick :innocent: 

 `protocolUpperCase`

Go formatting style prefers camelCase names.
Yes, I know we have other variables with an underscore like `api_v1` :smile: , but that is a special case.
Normal variables should be camelCase